AppKit/CPViewController.j

```diff
@@ -176,6 +176,58 @@ var CPViewControllerCachedCibs;
         _view = [CPView new];
 }
 
+- (void)loadViewAsynchronously:(Function/*(view, error)*/)aHandler
```

Comment:
Here I think we should using a name pattern on a bigger scale. I have used a pattern like ```loadViewWithCompletionBlock:(Function)completionBlock```. Some places uses a pattern like ```loadViewWithCompletionHandler:(Function)aHandler```. It should be good if we can come to some consensus about this. I think the word Asynchronously feels a little out of place but maybe I'm wrong...